src/components/donation/Steps/DonateMethods/Stripe/Checkout.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  PaymentElement,
+  useElements,
+  useStripe,
+} from "@stripe/react-stripe-js";
+import { FormEventHandler, useState } from "react";
+import { useErrorContext } from "contexts/ErrorContext";
+import LoadText from "components/LoadText";
+import { GENERIC_ERROR_MESSAGE } from "constants/common";
+import { appRoutes } from "constants/routes";
+
+// Code inspired by React Stripe.js docs, see:
+// https://stripe.com/docs/stripe-js/react#useelements-hook
+export default function Checkout({ onBack }: { onBack: () => void }) {
```

Comment:
after specifying amount and currency, the next screen is blank
<img width="689" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/f9a295f8-00d0-41b4-a8ee-1a0d8af2606a">

Reply:
Dear Lord, again... Looking into it!

Reply:
@ap-justin can't repro, can you post exact repro steps?

Also, did you pull the latest branch changes before testing? 
And did you run `yarn install` when switching to this branch? Please try `rm -rf node_modules && yarn install` and then test things out, maybe there occurred a corruption in your *node_modules*

Reply:
weird, I always delete local branch and pull latest remote. works now👍